components/aws/sagemaker/tests/integration_tests/conftest.py

```diff
@@ -86,12 +87,24 @@ def kfp_client():
     kfp_installed_namespace = utils.get_kfp_namespace()
     return kfp.Client(namespace=kfp_installed_namespace)
 
+def get_experiment_id(kfp_client):
+    exp_name = datetime.now().strftime("%Y-%m-%d-%H-%M")
+    experiment = kfp_client.create_experiment(name=exp_name)
```

Comment:
Keep the logic same as before it will fail if one run tests more than once in a minute

Reply:
This try/catch logic was actually breaking, for some reason. Perhaps the catch type did not match properly.